src/components/FormComponents/RhelAwsCheckbox.js

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+
+import { Checkbox } from '@patternfly/react-core';
+import { useFlag } from '@unleash/proxy-client-react';
+
+const RhelAwsCheckbox = ({ ...props }) => {
+  const { label, input } = useFieldApi(props);
+  const rhelAws = useFlag('platform.sources.metered-rhel');
+
+  return (
```

Comment:
Same as above. Check the flag and return `null` instead of wrapping the actual node in Fragment.